.github/workflows/build.yaml

```diff
@@ -0,0 +1,24 @@
+name: Deploy
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Deploy
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Compile the code
+      run: podman run --rm -v $(pwd):/app --workdir=/app golang:1.15 make build
```

Comment:
Not sure that librdkafka-dev 0.11.6 is available on this image. Please check it.